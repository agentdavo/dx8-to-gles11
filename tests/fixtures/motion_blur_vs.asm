#define CV_ZERO 0
#define CV_ONE 1
#define V_POSITION v0
#define V_DIFFUSE v1
#define V_TEXTURE v2

vs.1.1

mov oPos, V_POSITION
mov oT0, V_TEXTURE
mov oT1, V_TEXTURE
mov oT2, V_TEXTURE
mov oT3, V_TEXTURE
mov oD0, V_DIFFUSE
