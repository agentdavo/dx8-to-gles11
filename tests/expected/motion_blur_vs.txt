VERTEX_ATTRIB idx=0
MULTITEXCOORD4F stage=0
MULTITEXCOORD4F stage=1
MULTITEXCOORD4F stage=2
MULTITEXCOORD4F stage=3
COLOR4F
